code-of-conduct.md

```diff
@@ -154,6 +154,51 @@ The Woodcrest administration reserves the right to monitor and moderate all grou
 
 By implementing this group creation and approval process, Woodcrest aims to foster a diverse and engaging roleplay environment while maintaining consistency, quality, and adherence to our shared values and standards.
 
+### 15.0 Respect for Staff and Community Decisions
+
+All Woodcrest residents are expected to treat staff members and the community with respect, both in their interactions and when expressing opinions or disagreements. While constructive feedback and open dialogue are encouraged, it must be done in a respectful and professional manner.
+
+#### 15.1 Respectful Communication
+
+When voicing opinions or disagreements with staff decisions or community policies, residents should:
+- Use polite and respectful language
+- Avoid personal attacks, insults, or derogatory remarks
+- Focus on the issue at hand and provide constructive suggestions or alternative perspectives
+- Recognize that staff members are acting in the best interest of the entire community
+
+##### 15.1.1 OOC Communication and IC Actions
+
+The guidelines for respectful communication outlined in section 15.1 primarily apply to out-of-character (OOC) interactions, such as discussions on forums, Discord, or other community platforms.
+
+However, it is important to note that in-character (IC) actions and behaviors that are disrespectful, harassing, or inappropriate towards staff members or the community will be met with appropriate in-character consequences. This may include, but is not limited to:
+- IC legal repercussions, such as fines, arrests, or imprisonment
+- IC social consequences, such as damaged reputation, loss of trust, or exclusion from certain groups or events
+- IC economic consequences, such as loss of business, contracts, or partnerships
```

Comment:
```suggestion
- IC economic consequences, such as loss of contracts, or partnerships
```
Loss of business should always be an OOC decision. I'd probably want to reword this but suggesting to think of IC economic consequences that aren't just fines